src/main/webapp/sass/_core.scss

```diff
@@ -3637,4 +3637,27 @@ div.border {
 	button.btn {
 		text-transform: capitalize;
 	}
+}
+
+.FacetColumnResultValueWidget {
+	display: grid;
+	grid-template-columns: 25px auto 50px;
+	margin-right: 5px;
+	> * {
+		align-self: center;
+	}
+	.checkbox {
+		grid-column: 1 / span 1;
+	}
+	.value {
+		grid-column: 2 / span 1;
+		max-width: 355px;
```

Comment:
Does this need to be specified? The width is already constrained by the grid column, but maybe it's a grid quirk

Reply:
The grid column 2 is set to auto, so without this the grid will extend outside of the container boundary to render the facet value content.